lms/static/scripts/frontend_apps/components/test/BasicLtiLaunchApp-test.js

```diff
@@ -322,14 +323,13 @@ describe('BasicLtiLaunchApp', () => {
         'LaunchErrorDialog[errorState="canvas-file-not-found-in-course"]'
       );
 
-      // Click the "Try again" button and verify that files are re-fetched without re-authorizing.
-      fakeApiCall.resetHistory();
       act(() => {
         errorDialog.prop('onRetry')();
       });
-      assert.notCalled(FakeAuthWindow);
-      assert.called(fakeApiCall);
-      await spinnerVisible(wrapper);
+      assert.calledWith(FakeAuthWindow, {
+        authToken: 'dummyAuthToken',
+        authUrl,
+      });
```

Comment:
I think both tests may also want to simulate clicking through the authorization and then assert that the API call was re-tried after authorizing